### DISCUSSION TITLE
Exception: InitGameServer returned false

### DISCUSSION THREAD

**Discussion by Yuoko** [FakeFishGames/Barotrauma#15312] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I tried to install a dedicated server according to the instructions from "https://barotraumagame.com/wiki/Hosting_a_Dedicated_Server ", but the server throws an exception. I also tried to fix it according to the instructions, but it didn't help.

### Reproduction steps

1. follow the linux installation instructions on the website
2. try to start server

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Linux

### Relevant error messages and crash reports

```shell
Barotrauma Dedicated Server crash report (generated on 12/06/2024 23:03:15)


Barotrauma seems to have crashed. Sorry for the inconvenience! 


Game version 1.6.19.1 (ReleaseLinux, branch release/autumn-update-hotfix-2, revision 89f5cd1f61)
Language: English
Selected content packages: Vanilla (A47CDC1)
Level seed: no level loaded
Loaded submarine: None
Selected screen: None
Server (Round hadn't been started)


System info:
    Operating system: Unix 5.15.0.119 64 bit


Exception: InitGameServer returned false (0,0,27015,0,NoAuthentication,"1.0.0.0") (System.Exception)
Target site: Void Init(Steamworks.AppId, Steamworks.SteamServerInit, Boolean)
Stack trace: 
   at Steamworks.SteamServer.Init(AppId appid, SteamServerInit init, Boolean asyncCallbacks) in <DEV>\Libraries\Facepunch.Steamworks\SteamServer.cs:line 88
   at Barotrauma.Steam.SteamManager.CreateServer(GameServer server, Boolean isPublic) in <DEV>\Barotrauma\BarotraumaServer\ServerSource\Steam\SteamManager.cs:line 23
   at Barotrauma.Networking.GameServer.StartServer(Boolean registerToServerList) in <DEV>\Barotrauma\BarotraumaServer\ServerSource\Networking\GameServer.cs:line 223
   at Barotrauma.GameMain.StartServer() in <DEV>\Barotrauma\BarotraumaServer\ServerSource\GameMain.cs:line 244
   at Barotrauma.GameMain.Run() in <DEV>\Barotrauma\BarotraumaServer\ServerSource\GameMain.cs:line 297
   at Barotrauma.Program.Main(String[] args) in <DEV>\Barotrauma\BarotraumaServer\ServerSource\Program.cs:line 85


Last debug messages:
   12/06/2024 23:03:15 - [12/06/2024 23:03:15]
  Saving banlist
   12/06/2024 23:03:15 - [12/06/2024 23:03:15]
  Using Lidgren networking. Manual port forwarding may be required. If players cannot connect to the server, you may want to use the in-game hosting menu (which uses Steamworks and EOS networking and does not require port forwarding).
   12/06/2024 23:03:15 - [12/06/2024 23:03:15]
  Starting the server...
   12/06/2024 23:03:15 - Selected shuttle: Hemulen
```

**Comment by donjuanplatinum** [comment]
hello yuoko, you need to `mkdir -p ~/.steam/sdk64`

and then `cp Barotrauma\ Dedicated\ Server/linux64/steamclient.so ~/.steam/sdk64`

**Reply by Yuoko** [reply]
what if I don't have a .steam folder and only have Steam

**Reply by Yuoko** [reply]
![image](https://github.com/user-attachments/assets/17c2b83e-da1e-44ea-827e-875a7e50f662)
the file is in the folder, but the server still won't start

**Reply by donjuanplatinum** [reply]
create a ~/.steam/sdk64

**Reply by Yuoko** [reply]
> create a ~/.steam/sdk64

It didn't help
![image](https://github.com/user-attachments/assets/02dbc32f-4bce-4c9f-a9b7-a75b55ed8529)
![image](https://github.com/user-attachments/assets/35d43b33-4004-4a69-ad55-7176493bce1c)

**Reply by Yuoko** [reply]
what else can help solve the problem?

**Reply by donjuanplatinum** [reply]
do u use steamcmd to run

**Reply by Yuoko** [reply]
> do u use steamcmd to run
I use ./DedicatedServer
